drivers/pci/controller/vmd.c

```diff
@@ -132,10 +164,10 @@ struct vmd_dev {
 	struct vmd_irq_list	*irqs;
 
 	struct pci_sysdata	sysdata;
-	struct resource		resources[3];
+	struct resource		resources[VMD_RESOURCE_COUNT];
 	struct irq_domain	*irq_domain;
-	struct pci_bus		*bus;
-	u8			busn_start;
+	struct pci_bus		*bus[VMD_ROOTBUS_COUNT];
+	u8			busn_start[VMD_ROOTBUS_COUNT];
```

Comment:
It is asking to be placed in structure.

```
struct vmd_pci_bus {
    struct pci_bus *bus;
    u8 busn_start;
```

```suggestion
struct vmd_pci_bus vmd_bus[VMD_ROOTBUS_COUNT];
```
but I think it is fine either way

Reply:
No changes here.